rustls/src/enums.rs

```diff
@@ -114,381 +114,381 @@ enum_builder! {
-        TLS_DH_DSS_WITH_AES_256_CBC_SHA256 => 0x0068,
-        TLS_DH_RSA_WITH_AES_256_CBC_SHA256 => 0x0069,
-        TLS_DHE_DSS_WITH_AES_256_CBC_SHA256 => 0x006a,
-        TLS_DHE_RSA_WITH_AES_256_CBC_SHA256 => 0x006b,
-        TLS_DH_anon_WITH_AES_128_CBC_SHA256 => 0x006c,
-        TLS_DH_anon_WITH_AES_256_CBC_SHA256 => 0x006d,
-        TLS_DHE_DSS_WITH_3DES_EDE_CBC_RMD => 0x0072,
-        TLS_DHE_DSS_WITH_AES_128_CBC_RMD => 0x0073,
-        TLS_DHE_DSS_WITH_AES_256_CBC_RMD => 0x0074,
-        TLS_DHE_RSA_WITH_3DES_EDE_CBC_RMD => 0x0077,
-        TLS_DHE_RSA_WITH_AES_128_CBC_RMD => 0x0078,
-        TLS_DHE_RSA_WITH_AES_256_CBC_RMD => 0x0079,
-        TLS_RSA_WITH_3DES_EDE_CBC_RMD => 0x007c,
-        TLS_RSA_WITH_AES_128_CBC_RMD => 0x007d,
-        TLS_RSA_WITH_AES_256_CBC_RMD => 0x007e,
-        TLS_GOSTR341094_WITH_28147_CNT_IMIT => 0x0080,
-        TLS_GOSTR341001_WITH_28147_CNT_IMIT => 0x0081,
-        TLS_GOSTR341094_WITH_NULL_GOSTR3411 => 0x0082,
-        TLS_GOSTR341001_WITH_NULL_GOSTR3411 => 0x0083,
-        TLS_RSA_WITH_CAMELLIA_256_CBC_SHA => 0x0084,
-        TLS_DH_DSS_WITH_CAMELLIA_256_CBC_SHA => 0x0085,
-        TLS_DH_RSA_WITH_CAMELLIA_256_CBC_SHA => 0x0086,
-        TLS_DHE_DSS_WITH_CAMELLIA_256_CBC_SHA => 0x0087,
-        TLS_DHE_RSA_WITH_CAMELLIA_256_CBC_SHA => 0x0088,
-        TLS_DH_anon_WITH_CAMELLIA_256_CBC_SHA => 0x0089,
-        TLS_PSK_WITH_RC4_128_SHA => 0x008a,
-        TLS_PSK_WITH_3DES_EDE_CBC_SHA => 0x008b,
-        TLS_PSK_WITH_AES_128_CBC_SHA => 0x008c,
-        TLS_PSK_WITH_AES_256_CBC_SHA => 0x008d,
-        TLS_DHE_PSK_WITH_RC4_128_SHA => 0x008e,
-        TLS_DHE_PSK_WITH_3DES_EDE_CBC_SHA => 0x008f,
-        TLS_DHE_PSK_WITH_AES_128_CBC_SHA => 0x0090,
-        TLS_DHE_PSK_WITH_AES_256_CBC_SHA => 0x0091,
-        TLS_RSA_PSK_WITH_RC4_128_SHA => 0x0092,
-        TLS_RSA_PSK_WITH_3DES_EDE_CBC_SHA => 0x0093,
-        TLS_RSA_PSK_WITH_AES_128_CBC_SHA => 0x0094,
-        TLS_RSA_PSK_WITH_AES_256_CBC_SHA => 0x0095,
-        TLS_RSA_WITH_SEED_CBC_SHA => 0x0096,
-        TLS_DH_DSS_WITH_SEED_CBC_SHA => 0x0097,
-        TLS_DH_RSA_WITH_SEED_CBC_SHA => 0x0098,
-        TLS_DHE_DSS_WITH_SEED_CBC_SHA => 0x0099,
-        TLS_DHE_RSA_WITH_SEED_CBC_SHA => 0x009a,
-        TLS_DH_anon_WITH_SEED_CBC_SHA => 0x009b,
-        TLS_RSA_WITH_AES_128_GCM_SHA256 => 0x009c,
-        TLS_RSA_WITH_AES_256_GCM_SHA384 => 0x009d,
-        TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 => 0x009e,
-        TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 => 0x009f,
-        TLS_DH_RSA_WITH_AES_128_GCM_SHA256 => 0x00a0,
-        TLS_DH_RSA_WITH_AES_256_GCM_SHA384 => 0x00a1,
-        TLS_DHE_DSS_WITH_AES_128_GCM_SHA256 => 0x00a2,
-        TLS_DHE_DSS_WITH_AES_256_GCM_SHA384 => 0x00a3,
-        TLS_DH_DSS_WITH_AES_128_GCM_SHA256 => 0x00a4,
-        TLS_DH_DSS_WITH_AES_256_GCM_SHA384 => 0x00a5,
-        TLS_DH_anon_WITH_AES_128_GCM_SHA256 => 0x00a6,
-        TLS_DH_anon_WITH_AES_256_GCM_SHA384 => 0x00a7,
+        TLS_RSA_WITH_NULL_MD5 => 0x0001 NoDebug,
```

Comment:
Don't love the random-looking identifier in here. Did you consider an attribute (with the downside of adding an extra line for each `NoDebug` variant) or maybe some comment-like syntax?

Reply:
I have tried an alternative syntax which calls back to negative trait impls. Still looks quite weird though, but at least it's now clear which items are included and excluded in `Debug::fmt`